pkg/kn/commands/source/apiserver/flags.go

```diff
@@ -95,9 +114,9 @@ func (f *APIServerSourceUpdateFlags) Add(cmd *cobra.Command) {
 		`The mode the receive adapter controller runs under:,
 "Ref" sends only the reference to the resource,
 "Resource" send the full resource.`)
-	cmd.Flags().StringSliceVar(&f.Resources,
+	cmd.Flags().StringArrayVar(&f.Resources,
 		"resource",
```

Comment:
Why does the name `resource` not match the variable `f.Resources`? Should this not be `"resources"`?

Reply:
`resource` is a flag. `Resources` is an array of `resource`.
User uses `--resource abc --resource def` to set multiple resources, which will be saved in `f.Resources`.